src/interfaces/IVault.sol

```diff
@@ -5,14 +5,14 @@ interface IVault {
     // ------------------------------------------------ CUSTOM ERROR ------------------------------------------------
     error ZeroAddress();
     error UnrecognizedParam();
-    error BadHealthFactor();
```

Comment:
can we still keep the name HealthFactor? Much clearer

Reply:
I actually think collateral ratio is much clearer as it's easy to understand that if it's higher than the liquidation threshold then it's a bad value. Health factor would need a new reader to look for what that means first, it's a whole new phrase not used anywhere else in the codebase too.